rules/linux/lnx_install_root_certificate.yml

```diff
@@ -0,0 +1,27 @@
+title: Install Root Certificate
+id: 78a80655-a51e-4669-bc6b-e9d206a462ee
+description: Detects installed new certificate
+author: Ömer Günal, oscd.community
+date: 2020/10/05
+tags:
+    - attack.defense_evasion
+    - attack.t1553.004
+level: low
+logsource:
+    product: linux
+detection:
+    selection:
+        - CommandLine|contains|all:
+          - 'mv '
+          - '/usr/local/share/ca-certificates'
+    selection2:
+        - ProcessName|contains:
```

Comment:
I would rather use `ProcessName|endswith: '/update-ca-certificates'`, as it's more efficient and accurate (:

Reply:
Thank you very much, @omergunal !